config.json

```diff
@@ -78,6 +78,15 @@
       ]
     },
     {
+      "uuid": "6pol4yme-6665-4ri6-sl21-4651u11cb4a4",
```

Comment:
I can't help but notice that this has letters outside of a-f, not sure that's acceptable for a UUID. Also need to check https://github.com/exercism/haskell/pull/601#discussion_r144734644.

Reply:
Do I make another PR for the complex-numbers UUID, or I include it in here?

Reply:
As long as it's in a separate commit, I would take it either way, but I suggest a different PR since I can merge that one quickly and can't merge this one quickly yet.